### ISSUE TITLE
We need to be able to show / hide building or road classes on the map

### ISSUE THREAD

**Issue by timlinux** [inasafe/inasafe-fba#25]
In the chart we need a checkbox to the right of each label that when checked will hide/show buildings or roads of that class

![image](https://user-images.githubusercontent.com/178003/72504371-9c9a4700-3835-11ea-9138-1ea9caaeae3c.png)

If we cant put the widgets directly in the chart, propose a different user experience that is intuitive

**Comment by ann26** [comment]
hi @NyakudyaA could you give me the example of cql filter for building type and road type, please?

**Comment by NyakudyaA** [comment]
@ann26 
&cql_filter=roads_id IN (2,13) etc
where roads id corresponds to those types. Do not include the type false.
![id-roads](https://user-images.githubusercontent.com/2510900/73719803-22b8f780-4729-11ea-8fd5-c40e11f94f09.png)

&cql_filter=building_id in (10,3) etc 
![building-id](https://user-images.githubusercontent.com/2510900/73720493-bc34d900-472a-11ea-9902-e585ae488cc0.png)

**Comment by ann26** [comment]
hi @NyakudyaA I tried this url: 
```
http://staging.fbf.kartoza.com/geoserver/wms?service=WMS&request=GetMap&layers=kartoza:exposed_buildings&styles=&format=image/png&transparent=true&version=1.1.1&srs=EPSG:3857&cql_filter=building_id IN (3,4,5,6,7,8,9,10,11,12,13,14) AND flood_event_id=212 AND village_id=3603081006 AND depth_class=3&width=256&height=256&bbox=11871587.737027295,-705666.6451287476,11872810.729479859,-704443.6526761842
```
but it doesn't work, no exposed buildings are shown. I included all available building ids

**Comment by NyakudyaA** [comment]
> `http://staging.fbf.kartoza.com/geoserver/wms?service=WMS&request=GetMap&layers=kartoza:exposed_buildings&styles=&format=image/png&transparent=true&version=1.1.1&srs=EPSG:3857&cql_filter=building_id IN (3,4,5,6,7,8,9,10,11,12,13,14) AND flood_event_id=212 AND village_id=3603081006 AND depth_class=3&width=256&height=256&bbox=11871587.737027295,-705666.6451287476,11872810.729479859,-704443.6526761842`

Hi @ann26 it seems the building_id that is in that column is not the correct one. I have modified it to call it `building_class_id` can you try it. Please let me know if it is the same with roads

**Comment by ann26** [comment]
yes sorry @NyakudyaA  I checked again after writing my comment, I use building_type and that was working, I will try that too. Thank you.

**Comment by NyakudyaA** [comment]
> yes sorry @NyakudyaA I checked again after writing my comment, I use building_type and that was working, I will try that too. Thank you.

yes, building type will work as well, but I think to increase the efficiency of a search it's better to do it against the number. The difference might be so small in the context of the kind of data we are showing
